File: sort_associative_array/sort_associative_array.pl
use strict;
my $example_array = {1 => 'Janualry', 12 => 'December', 3 => 'March' };
foreach my $key (sort {$a <=> $b} keys %{$example_array}) {
   print $key. " => " . $example_array->{$key} ."\n";
}

File: sort_associative_array/sort_associative_array.py
import collections
example_array = {1: "Janualry", 12:"December", 3:"March" }
example_array2 = collections.OrderedDict(sorted(example_array.items(), key=lambda t: t[0]))
for key, value in example_array2.items():
  print(str(key) + " => " + value)

